games/devtest/mods/testhud/init.lua

```diff
@@ -206,9 +206,72 @@ minetest.register_chatcommand("zoomfov", {
 	end,
 })
 
+-- Hotbars
+
+local hud_hotbar_defs = {
+	{
+		type = "hotbar",
+		position = {x=0.2, y=0.5},
+		direction = 0,
+		alignment = {x=0, y=-1},
+	},
+	{
+		type = "hotbar",
+		position = {x=0.2, y=0.5},
+		direction = 2,
+	},
+	{
+		type = "hotbar",
+		position = {x=0.7, y=0.5},
```

Comment:
That's a bug which the hotbar inherits from the `inventory` hud element type.
It happens when the inventory gets drawn outside off the screen.

I think it would make more sense to fix it in a separate PR, since this PR is already big enough, and isn't a problem for the predefined builtin hotbar. (Like #14318)